modules/rostests/apitests/shell32/CMakeLists.txt

```
@@ -37,6 +37,7 @@ list(APPEND SOURCE
     ShellExecCmdLine.cpp
     ShellExecuteEx.cpp
     ShellExecuteW.cpp
+    ShellInfo.cpp
     ShellHook.cpp
```

```suggestion
    ShellHook.cpp
    ShellInfo.cpp
```
(alphabetical order)